x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/index.ts

```diff
@@ -20,10 +18,4 @@ export type {
   EntityNodeViewModel,
   NodeProps,
 } from './types';
-
-export const isEntityNode = (node: NodeViewModel) =>
-  node.shape === 'ellipse' ||
-  node.shape === 'pentagon' ||
-  node.shape === 'rectangle' ||
-  node.shape === 'diamond' ||
-  node.shape === 'hexagon';
+export { isEntityNode } from './utils';
```

Comment:
The previous code was causing a circular dependency when it was imported by `GraphInvestigation`

Reply:
Interesting, I don't see how moving it to another file but keeping the export on `index.ts` fixes the circular dependency, but moving to another file was a good practice.